plugin/evm/message/codec.go

```diff
@@ -41,6 +43,10 @@ func init() {
 		c.RegisterType(CodeRequest{}),
 		c.RegisterType(CodeResponse{}),
 
+		// p2p sdk gossip types
+		c.RegisterType(gossip.PullGossipRequest{}),
+		c.RegisterType(gossip.PullGossipResponse{}),
```

Comment:
should we add these to a new codec (for the p2p sdk -- I wonder if some earlier types referenced here would cause it to be more difficult to migrate to a different codec later?)

Reply:
I think that makes sense. I added a separate codec for the sdk types.